homeassistant/components/logbook.py

```diff
@@ -286,6 +287,28 @@ def humanify(hass, events):
                     'context_user_id': event.context.user_id
                 }
 
+            elif event.event_type == EVENT_HOMEKIT_CHANGED:
+                data = event.data
+                entity_id = data.get(ATTR_ENTITY_ID)
+                message = data.get('message')
+
+                if message:
+                    action = message
+                elif entity_id:
+                    action = "Send command for {}".format(
+                        entity_id)
+
+                yield {
+                    'when': event.time_fired,
+                    'name': 'HomeKit',
+                    'message': action,
+                    'domain': 'homekit',
+                    'entity_id': entity_id,
+                    'context_id': event.context.id,
+                    'context_user_id': event.context.user_id
+                }
+
+
```

Comment:
One empty line to much :)

Reply:
Not sure why this was not resolved as I did remove the extra line in there. Also checked the code with pylint, pydocstyle, and flake8 and no issues reported.
Will set this to resolve manually. :-)